cython/core/tail.pyx

```diff
@@ -184,11 +184,11 @@ def get_triangulation_tester():
         vol = M.volume()
         if abs(vol) < 0.1:
             vol = 0.0
-        print M, vol, M.homology()
+        print(M, vol, M.homology())
 
     for spec in specs:
         M = Triangulation(spec)
-        print M, M.homology()
+        print(M, M.homology())
```

Comment:
This is causing at least some of the CI failures: since Cython is in Py2 mode, `print a, b` is not equivalent to `print(a, b)`.